### PR TITLE
tightening up the win ci files

### PR DESCRIPTION
**Pull request by devoncarew**
Trying to make the windows CI files shorter and cleaner.

### CODE REVIEW

**Comment by devoncarew** [comment]
@guillermooo, I experimented with shrinking the amount of code needed to get appveyor up and running. It's awesome to have this available to dart projects now!

File: appveyor.yml [hunk 1]
```diff
@@ -3,9 +3,15 @@
 # license that can be found in the LICENSE file.
 
 install:
-  - ps: .\tool\appveyor.ps1 install
+  - cinst -y dart-sdk
+  - ps: $env:PATH = "$env:PATH;c:\tools\dart-sdk\bin"
+  - ps: $env:PATH = "$env:PATH;c:\Users\appveyor\AppData\Roaming\Pub\Cache\bin"
```
Comment (guillermooo) [inline appveyor.yml]: Maybe:

    $env:APPDATA\Pub\Cache\bin

File: appveyor.yml [hunk 2]
```diff
@@ -3,9 +3,15 @@
 # license that can be found in the LICENSE file.
 
 install:
-  - ps: .\tool\appveyor.ps1 install
+  - cinst -y dart-sdk
+  - ps: $env:PATH = "$env:PATH;c:\tools\dart-sdk\bin"
```
Comment (guillermooo) [inline appveyor.yml]: If we're mostly using CMD.exe, we could here too:

    SET PATH=%PATH%;C:\tools\dart-sdk\bin

File: appveyor.yml (hunk 1 above)
Comment (guillermooo) [inline appveyor.yml]: (BTW, I'm not sure why Dart puts scripts under the roaming profile, as dart may not be available on all computers that a given user uses.)

**Comment by guillermooo** [comment]
Looks great! I didn't know that the latest dart-sdk was available through Chocolatey now. Just commented a couple minor things.

File: appveyor.yml (hunk 1 above)
Comment (devoncarew) [inline appveyor.yml]: I'll update this (tha path). Not sure about the APPDATA usage; perhaps it's necessary for other reasons? Or, perhaps there was some unfamiliarity with windows conventions.

File: appveyor.yml (hunk 2 above)
Comment (devoncarew) [inline appveyor.yml]: done

**Comment by devoncarew** [comment]
Yup, Chocolatey has Dart SDK v1.8.5 right now. The maintainer just changed, but it is being updated.

File: appveyor.yml (hunk 1 above)
Comment (guillermooo) [inline appveyor.yml]: Maybe I'll repor to `pub` issues that `%LOCALAPPDATA%` exists on Windows just in case.

**Pull request merged**